yarn.lock

```diff
@@ -3468,19 +3468,6 @@
     ethereumjs-util "7.1.5"
     stack-trace "0.0.10"
 
-"@superfluid-finance/ethereum-contracts@1.4.3":
```

Comment:
what happened here? @kasparkallas

Reply:
It weirdly appeared here with a `yarn install`: https://github.com/superfluid-finance/protocol-monorepo/pull/1255#discussion_r1087562403 and now it disappeared again. :joy:

Reply:
packages r pain 🥲